app/view-plants/page.tsx

```diff
@@ -0,0 +1,110 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+import { UUID } from 'crypto';
+import supabase from '@/api/supabase/createClient';
+import { get_plant_by_id } from '@/api/supabase/queries/plant_by_id';
+import { getPlantSeasonality } from '@/api/supabase/queries/plantSeasonality';
+import PlantCard from '@/components/PlantCard/PlantCard';
+import { Plant } from '@/types/schema';
+
+export default function Home() {
+  const [selectedOption, setSelectedOption] = useState<'myPlants' | 'all'>(
+    'myPlants',
+  );
+  const [isSelected, setIsSelected] = useState<true | false>(false);
+
+  const [plants, setPlants] = useState<Plant[]>([]);
+  const [userPlants, setUserPlants] = useState<Plant[]>([]);
+  const user_id: UUID = 'e72af66d-7aae-45f6-935a-187197749d9f';
+  async function fetchUserPlants(user_id: UUID) {
+    const { data, error } = await supabase
+      .from('user_plants')
+      .select('plant_id')
+      .eq('user_id', user_id);
+
+    if (error) {
+      console.error('Error fetching plant IDs:', error);
+      return [];
+    }
+
+    const plantIds = data?.map(row => row.plant_id) || [];
+    if (plantIds.length === 0) {
+      return [];
+    }
+
+    const plantsUser: Plant[] = await Promise.all(
+      plantIds.map(plantId => get_plant_by_id('Tennessee', plantId)),
```

Comment:
when I console.log'd, this function didn't seem to be working 
even tho there was a user_plant row, the `plantsUser` was returning as `false`
<img width="182" alt="Screen Shot 2024-10-18 at 9 08 11 PM" src="https://github.com/user-attachments/assets/35403a1a-66e9-49df-a532-f056d31b1338">